README.md

```diff
@@ -20,10 +20,10 @@ Creating a trust-minimised distributed Valdator cluster requires a multi-sig wal
 
 ## The SSV client
 
-`ssvnode` is the software that allows validators to be run on a group of independent nodes - a cluster. A complete multi-container `Docker` setup including execution client, consensus client, MEV-Boost and the `ssvnode` client can be found in this repository https://github.com/cnupy/ssv-validator-node and the first step is to clone it:
+`ssvnode` is the software that allows validators to be run on a group of independent nodes - a cluster. A complete multi-container `Docker` setup including execution client, consensus client, MEV-Boost and the `ssvnode` client can be found in this ETH Docker repository https://github.com/eth-educators/eth-docker and the first step is to clone it:
 
 ```
-git clone https://github.com/cnupy/ssv-validator-node.git
+cd ~ && git clone https://github.com/eth-educators/eth-docker.git && cd eth-docker
 ```
 
 Make sure your user has the `docker` role. If not you can use this command to add it:
```

Comment:
This is unnecessary, as eth-docker adds user to the docker group during installation.

https://github.com/eth-educators/eth-docker/blob/f49414256359d6410d7483e0cb90a90cbc0735c2/ethd#L326

```suggestion
```